UnitTests/Dialogs/DialogTests.cs

```diff
@@ -1331,4 +1331,33 @@ public void Zero_Buttons_Works ()
 
         return (Begin (dlg), dlg);
     }
+
+    [Fact]
+    [SetupFakeDriver]
+    [TestRespondersDisposed]
+    public void Run_Does_Not_Dispose_Dialog ()
```

Comment:
Did you realized that running this unit test alone never ends because it's using `WindowsDriver` and not `FakeDriver`. Also the `Ready` event isn't invoked because it's called after the `if (MainLoop.Running && MainLoop.EventsPending ())` where it never get out.